docs/build/_partials/_oracles_contract_data.md

```diff
@@ -0,0 +1,37 @@
+import { AddToMetaMaskButton } from '@theme/AddToMetaMaskButton';
+import { Networks } from '@theme/constant';
```

Comment:
Why is this here? I don't see it used. Also shouldn't this be an MDX file, because we are importing?

Reply:
Yes and yes

Reply:
Fixed that in another [PR](https://github.com/iotaledger/iota-wiki/pull/1604)